deploy/charts/openebs-monitoring/values.yaml

```diff
@@ -165,6 +165,8 @@ alertRules:
     enabled: true
   Jiva:
     enabled: true
+  volumes:
```

Comment:
q_: what is this used for?

Reply:
It is required for making PrometheusRule object for the rules defined in that directory.

Reply:
got it. Thanks!